Framework/Geometry/test/MeshObjectTest.h

```diff
@@ -1105,7 +1105,48 @@ class MeshObjectTestPerformance : public CxxTest::TestSuite {
     }
   }
 
+  void test_solid_angle() {
+    const size_t dim(20);
+    V3D test_pt;
+    double dummy;
+    for (size_t i = 0; i < dim*dim*dim; ++i) {
```

Comment:
👍 For adding important performance tests to this performance test class. However, the `create_test_point` calls are going to pollute the performance test timings. We are only interested in the `MeshObject` methods. Can you move the creation of the test points to outside the loops in these test methods so that they have minimal impact, or ideally create the test point in the `MeshObjectTestPerformance` constructor?

Reply:
OK I will deal with that. 

I'll be away from tonight till Tuesday. Also after attempting to rebase and fix a merge conflict my branch has become unpushable and I will be dealing with that when I come back. Then I'll respond to this and any other reviews you make while I am away.